config/initializers/redis.rb

```diff
@@ -0,0 +1 @@
+$REDIS_CLIENT = Redis.new url: Settings.redis.database.uri
```

Comment:
This is the most important change: 1 global constant defined for 1 connection to Redis (this is used in the "database" redis, i.e., where we store temporary values returned from SC to be updated in the browser and then deleted. This is the one thing that was making heroku fail I think.